sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/WebJobsEventHubTestBase.cs

```diff
@@ -18,7 +16,7 @@ namespace Microsoft.Azure.WebJobs.Host.EndToEndTests
     public class WebJobsEventHubTestBase
     {
         protected const string TestHubName = "%webjobstesthub%";
-        protected const int Timeout = 30000;
+        protected static readonly int Timeout = (int)EventHubsTestEnvironment.Instance.TestExecutionTimeLimit.TotalMilliseconds;
```

Comment:
What is this value?

Reply:
To normalize with the other Event Hubs tests and fix the intermittent timeout issue that is causing flakiness today.

Reply:
I don't think we want to change it to 3 minutes, or 10 minutes. Right now it is set to 30 seconds.

Reply:
Was the flakiness due to timing out?

Reply:
Yes; to my knowledge, the flakiness is due to timeouts.   Local runs were fine, CI runs were hit-or-miss.   I figured that aligning things to the rest of the Event Hubs ecosystem makes the most sense, since it's using the same test infrastructure and Azure resources.

Reply:
Confirmed offline that the tests are non-parallel.   Dropped the timeout to 60 seconds, since the concurrent behavior in CI isn't an issue.